src/main/java/org/opentripplanner/updater/alerts/AlertsUpdateHandler.java

```diff
@@ -87,11 +85,9 @@ private TransitAlert mapAlert(String id, GtfsRealtime.Alert alert) {
                 routeId = informed.getRouteId();
             }
 
-            int direction;
-            if (informed.hasTrip() && informed.getTrip().hasDirectionId()) {
-                direction = informed.getTrip().getDirectionId();
-            } else {
-                direction = -1;
+            int directionId = MISSING_INT_FIELD_VALUE;
+            if (informed.hasDirectionId()) {
```

Comment:
Should we also handle informed.getTrip().getDirectionId() here. I'm not quite sure what is the difference between them other than that the trip descriptor's direction was added to the spec first(?). I have a feeling that the trip descriptor's directionId should be used for fuzzy trip matching to find individual trips instead of using it just together with the route_id.

Reply:
We decided that the TripDescriptor's direction_id should only be used for fuzzy matching.